src/main/java/at/ac/tuwien/kr/alpha/grounder/structure/AnalyzeUnjustified.java

```diff
@@ -298,13 +298,13 @@ public Atom next() {
 		}
 	}
 
-
 	private List<RuleAndUnifier> rulesHeadUnifyingWith(Atom p) {
 
 		List<RuleAndUnifier> rulesWithUnifier = new ArrayList<>();
 		Predicate predicate = p.getPredicate();
 		// Check if literal is built-in with a fixed interpretation.
-		if (p instanceof FixedInterpretationLiteral) {
+		// if (p instanceof FixedInterpretationLiteral) {
```

Comment:
This looks dangerous. I do not think it is correct. What is the rationale for the change?